roles/edpm_nova/molecule/default/prepare.yml

```diff
@@ -178,12 +178,12 @@
         group: "{{ item.group | default(ansible_user) }}"
         mode: "{{ item.mode | default(omit) }}"
       loop:
-        - {"path": "/var/lib/openstack/config/nova", "mode": "0755"}
```

Comment:
this kind of break the intended stureu fo this folder

service config were ment to be in the config folder so that we could have cert and other sibling folder that were not used for service config.